tox.ini

```diff
@@ -1,44 +1,59 @@
 [tox]
-envlist = py27,py3{6,7,8,9,10,11},flake8,docs
-skip_missing_interpreters = true
+envlist = coverage_clean, py27, py38, py3, coverage_report, flake8
 isolated_build = true
 skipsdist = true
 
-[flake8]
-max-line-length = 99
-
+# Unit tests
 [testenv]
+setenv =
+    tests_dir = {toxinidir}{/}tests/unit
+    PYTHONPATH = {toxinidir}
+commands =
+    python -m pytest --cov-append --cov-report=xml --cov=ssg "{env:tests_dir}{/}ssg-module"
+
+deps =
+    pyyaml
+    Jinja2
+    -r test-requirements.txt
+
+# Coverage
+[testenv:coverage_report]
+deps = coverage
 skip_install = true
-install_command = pip install {opts} {packages}
+commands =
+    coverage html
+    coverage xml
+    coverage report -m
+
+[testenv:coverage_clean]
+deps = coverage
+skip_install = true
+commands = coverage erase
+
+
+# Code style
+[flake8]
+max-line-length = 99
+per-file-ignores =__init__.py:F401
```

Comment:
A comment on why would be a good idea.

Reply:
In the new OVAL object model, imports are specified in __init__.py files to make it easier to use subpackages. These imports are therefore not used in these files.